auacm/app/database.py

```diff
@@ -3,14 +3,23 @@
 from sqlalchemy.orm import Session
 from sqlalchemy.ext.automap import automap_base
 
-# database setup
+from app import app
+
+# Use the test DB if specified, otherwise the default
 Base = automap_base()
 engine = create_engine('mysql+pymysql://acm@localhost/acm?charset=utf8')
 connection = engine.connect()
 session = Session(engine)
+
+# Set up the test database connection
+test_engine = create_engine(
```

Comment:
Yes. Not ideal, I'll admit it, but it's the best I could come up with. This get called from app/__init__ so there's no way to change the config programmatically before the connection is established. This is also in part a result of the global variable madness. It can be changed, I think.